src/plugins/intel_cpu/src/emitters/plugin/x64/jit_bf16_emitters.hpp

```diff
@@ -33,56 +38,115 @@ class jit_uni_vcvtneps2bf16 : public jit_emitter {
         }
     }
 
+    template <typename Vmm>
+    void saturate_input(Vmm& truncated,
+                        const Vmm& in,
+                        const std::string& bf16_min_key,
+                        const std::string& bf16_max_key) const {
+        Vmm bf16_max = Vmm(aux_vec_idxs[0]);
+        Vmm bf16_min = Vmm(aux_vec_idxs[1]);
+        h->uni_vmovups(bf16_max, table_val(bf16_max_key));
+        h->uni_vmovups(bf16_min, table_val(bf16_min_key));
+        h->uni_vmaxps(truncated, in, bf16_min);
```

Comment:
I still think "truncated" name in saturation function is fusing, especially we note them different modes. Could we use "clamped" or smt?

Reply:
changed name to "clamped"